network-mux/src/Network/Mux/Trace.hs

```diff
@@ -49,9 +49,9 @@ data MuxErrorType = MuxUnknownMiniProtocol
                   | MuxIngressQueueOverRun
                   -- ^ thrown by 'demux' when violating 'maximumIngressQueue'
                   -- byte limit.
-                  | MuxControlProtocolError
-                  -- ^ thrown by 'muxControl' (mux control thread), when
-                  -- received a 'Muxcontrol' message on a mature 'MuxBearer'.
+                  | MuxInitiatorOnly
+                  -- ^ thrown when data arrives on a responder channel when the
+                  -- mux was set up as an 'InitiatorApp'.
```

Comment:
:+1: At least pre refactoring receiving a message on an initiator only bearer would have resulted in a `MuxUnknownMiniProtocol` error.

Reply:
I couldn't see how. The demuxer thread has a complete array of (protocol num, direction), with a queue for each one. It didn't know which queues had threads attached to them. The mux start would only start threads for the directions being used.

Reply:
That may have been have been during a previous iteration of my InititatorAndResponder work.